integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -278,6 +278,33 @@ void testAlterTableSetAndRemoveProperty() {
     Assertions.assertTrue(newProperties.containsKey("key2"));
   }
 
+  @Test
+  void testAlterTableAddAndDeleteColumn() {
+    String tableName = "test_column";
+    dropTableIfExists(tableName);
+
+    createSimpleTable(tableName);
+    String[] oldColumnNames =
+        getTableInfo(tableName).getColumns().stream()
+            .map(SparkColumnInfo::getName)
+            .toArray(String[]::new);
+    Assertions.assertEquals(oldColumnNames, new String[] {"id", "name", "age"});
+
+    sql(String.format("ALTER TABLE %S ADD COLUMNS (col1 string)", tableName));
+    String[] addColumnNames =
+        getTableInfo(tableName).getColumns().stream()
+            .map(SparkColumnInfo::getName)
```

Comment:
beside names, we should also check check column properties ,  suggest to check with SparkTableInfoChecker , you could create a specific table column, please refer `getSimpleTableColumn`.

Reply:
added column should be `SparkColumnInfo.of("col1", DataTypes.StringType, null)`